polaris.shopify.com/src/components/IconGrid/IconGrid.module.scss

```diff
@@ -23,6 +23,11 @@
     border-radius: var(--border-radius-600);
     box-shadow: var(--card-shadow);
     user-select: none;
+    color: var(--text);
```

Comment:
Thanks for removing that hard coded text color! A leftover from prototyping...

I wonder if we could use text-subdued for these, if we upped text-subdued to `#767676`?

<img width="256" alt="image" src="https://user-images.githubusercontent.com/875708/177199532-ef63a8b2-843f-4e37-aeb2-c96e94e9170c.png">